randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -150,7 +150,7 @@ Extra - asset_id: collision_camera_003
   * Extra - actor_name: dooremmy_002
   * Extra - actor_def: actordef:actors/props/dooremmy/charclasses/dooremmy.bmsad
   > Tunnel to Purple EMMI Introduction
-      Gravity Suit or Speed Booster
+      Gravity Suit or Speed Booster or Perform WBJ
```

Comment:
Can add Space Jump at least. Will need to see if Spin Boost is enough

Reply:
Spin boost is enough.. But it´s super tight.